lib/widgets/qpi/buttons.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+class LongButton extends StatefulWidget {
+  final String text;
+  final Color buttonColor;
+  final Color textColor;
+LongButton(this.text, this.buttonColor, this.textColor); 
+
+  @override
+  _LongButtonState createState() => _LongButtonState();
+}
+
+class _LongButtonState extends State<LongButton> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 219,
+      height: 56,
+      decoration: BoxDecoration(
+          color: widget.buttonColor,
+          borderRadius: BorderRadius.all(Radius.circular(5))),
```

Comment:
i'm not sure if `5` is the border (and with the width of `219` rin), most should be a factor of 8 already but clarify na lang with designers